catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -91,8 +91,8 @@
     </service>
 
 
-    <cm:managed-service-factory id="Src_Action_Provider"
-                                factory-pid="Src_Action_Provider" interface="ddf.action.ActionProvider">
+    <cm:managed-service-factory id="action.provider.src"
+                                factory-pid="action.provider.src" interface="ddf.action.ActionProvider">
```

Comment:
Typically we use the fully-qualified class name as the pid.

Reply:
Fixed.